### ISSUE TITLE
[A11Y] Output `Congratulations` for screenreader

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#3907]
Output `Congratulations` for screenreader.

Whenever a user submits a correct answer we show a `Congratulations` toast. This is currently not readable by screen-reader. 

It should be readable by screen-reader as soon as it appears.

**Comment by shubhamji88** [comment]
Could I refer: https://developer.android.com/codelabs/kotlin-android-training-design-for-everyone#4  
to know more about this, then proceed or can you suggest some other implementation.

**Comment by shubhamji88** [comment]
please assign it to me, i found that the congratulation message is a text view not a toast which is INVISIBLE in question_player_fragment.xml and to show congratulation it we are just making it VISIBLE with animations along with KonfettiView.
is there anything else that i am missing ?

**Comment by rt4914** [comment]
@shubhamji88 Actually `Congratulations` is shown in `question_player_fragment.xml` as well as `state_player_fragment.xml or exploration_player_fragment (one of these)`.

We don't want to change the UI. We just want to make sure that as soon as the `Congratualtions` is displayed the `Talkback app` or any other screen-reader can read it. Currently `Talkback` is unable to read it.

@shubhamji88 Before assign you this issue, can you post an idea on how are you trying to solve this?

**Comment by shubhamji88** [comment]
i tried to use visibility GONE and VISIBLE but it didn't worked...thinking of some other solution
